code/object/collideshipship.cpp

```diff
@@ -473,6 +473,33 @@ static bool check_subsystem_landing_allowed(ship_info *heavy_sip, collision_info
 	return false;
 }
 
+/**
+ * Helper function that calculates the next time this ship should be allowed to collide with a client player on multiplayer.
+ */
+int calculate_next_multiplayer_client_collision_time(float impulse_magnitude)
+{
+	// Uses a continuous cubic rational function that allows us to space out collisions based on collision strength.
+	static constexpr float IMPULSE_RANGE_FACTOR = 1.0f / powf(9900, 3.0f); // based on some quick tests from Asteroth about how strong an im
+	static constexpr float MIN_IMPULSE = 100;
+	static constexpr int LONGEST_COLLISION_INTERVAL = 200;
```

Comment:
These shouldn't be declared `static`. That allocates them on the global stack, where they have no business being.

Reply:
Should be fixed.